Django_HW/create/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.urls import path
+from . import views
+
+app_name = 'create'
+
+urlpatterns = [
+    path('', views.CreateListView.as_view(), name='create_list'),
```

Comment:
nit: convention about naming paths is using `-`
```suggestion
    path('', views.CreateListView.as_view(), name='create-list'),
```